### PR TITLE
Create 1908-01-13.xml

### PR DESCRIPTION
**Pull request by ak18f**
creating page 1

### CODE REVIEW

**Comment by whanley** [comment]
Is there really only one ad in the first column, and only one in the second?

**Comment by ak18f** [comment]
Hi,

I messed up my microfilm and first deliverable pretty badly, hence why I only had one ad on the first two columns and the rest on the third. I pull requested a new document redoing and completing the first page as well as my second and third page. Do I gain back full credit?

Thank You
________________________________

**Comment by ak18f** [comment]
Hi,

I'm having trouble doing the pull request, for some reason even though I'm fetching the organ through GitHub Desktop, it isn't giving me the option to pull request.

Thank You
________________________________

**Comment by whanley** [comment]
Looks good, Austin. A few things:
- at line 2426, use a `</div>` to close page 4 before you start page 5
- looks like your OCR is working well, but you have not formatted all of the paragraphs and titles. Check the tutorial, where I suggest how you can do this with regular expressions. Ask if you need help
- use templates for templated material--for instance Reuter's Telegram (the last div on page 6). Again, ask if you are unclear.

**Comment by whanley** [comment]
There are mistakes in your header--fix the dates on line 26 of both files.

You've done a nice job filling out these two issues with OCRed text and templates, but both need updating.

The OCR needs to be divided into sections, items, headlines, and paragraphs. Where appropriate, the sections should have feature attributes.

The templates need to be updated with the contents that appear in your issues of the paper.

File: 1908-01-18.xml [hunk 1]
```diff
@@ -0,0 +1,7608 @@
...
+                            <cell><measure quantity="8881" unit="ton">8881</measure> Tons</cell>
+                            <cell>about <date when="1905-08-14">14 August</date></cell>
+                            
+                        </row>
+                        <row>
+                            <cell><name>Gera</name></cell>
+                            <cell><measure quantity="5005" unit="ton">5005</measure> Tons</cell>
+                            <cell>about <date when="1905-07-31">31 July</date></cell>
+                            <cell><name>Pr. Regt. Luitpold</name></cell>
+                            <cell><measure quantity="6288" unit="ton">6288</measure> Ton
```
Comment (huoyanzhizhou) [inline 1908-01-18.xml]: Did you make up the xml id of deg-ad-cst01 here? My Oxygen give me a red box because of it.

**Comment by huoyanzhizhou** [comment]
Lots of new good work, just remember to use more features and update your measures!

**Comment by whanley** [comment]
merging these files

**Comment by whanley** [comment]
Lots of cleaning, as we discussed in class, but the core contents are here.

**Pull request merged**